src/main/java/com/google/cloud/discotoproto3converter/proto3/DocumentToProtoConverter.java

```diff
@@ -59,6 +59,15 @@ public String getPackageName() {
     return Collections.unmodifiableMap(allResourceOptions);
   }
 
+  private ProtoFile readDocumentMetadata(Document document, String documentFileName) {
+    return new ProtoFile(
+        documentFileName,
+        document.name(),
+        document.version(),
+        document.revision(),
+        "google.cloud." + document.name() + "." + document.version());
```

Comment:
As per the comment in the design doc, we don't always want to hard-code this to `cloud`. How about adding a TODO here to remind us to come back to that?

Reply:
Added TODO